webtoolbar.py

```diff
@@ -426,6 +437,19 @@ def __init__(self, tabbed_view, act):
 
         self._configure_toolbar()
 
+    def __download_running_cb(self):
+        '''
+        Updates the downloadIcon tooltip message and progress value.
+        '''
+        progress = downloadmanager.overall_downloads_progress()
+        self._download_icon.update(progress)
+        if progress > 0.0:
+            self._download_icon.set_tooltip(_("Downloading.. " +
```

Comment:
Just 1 thing before I can merge:  please use a format string here (eg "Downloading... {}%".format()), so that it is eaiser for the translator to eg, put the % before the downloading.